cmd/object-api-errors.go

```diff
@@ -76,7 +76,7 @@ func toObjectErr(err error, params ...string) error {
 				Object: params[1],
 			}
 		}
-	case errDataTooLarge:
+	case errDataTooLarge, errPolicyTooLarge:
```

Comment:
You don't need this change in this PR since this error is not used anywhere..

Reply:
Removed